modules/grafana/manifests/dashboards/deployment_dashboard.pp

```diff
@@ -117,6 +117,8 @@
     $elasticsearch_stats
   )
 
+  $sentry_environment = hiera('govuk::deploy::config::errbit_environment_name')
```

Comment:
We do not allow direct hiera lookups in this way (other than a few special snowflakes).

Set it as a parameter at the top of the manifest, and either:

1. Reference the variable in hieradata.
2. Reference the variable in the parameter, ie `$sentry_environment = $::govuk::deploy::config::errbit_environment_name`

Reply:
I think I've done what you suggest (option 2) and it passes lint.  Thanks @surminus!